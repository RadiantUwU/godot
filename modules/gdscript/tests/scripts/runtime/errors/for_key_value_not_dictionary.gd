func test():
    var a: Dictionary
    for k, v in a: # Runtime type check is not needed.
        pass
    var b: Variant = [1, 2, 3]
    for u, v in b:  # Runtime type check is needed.
        pass
