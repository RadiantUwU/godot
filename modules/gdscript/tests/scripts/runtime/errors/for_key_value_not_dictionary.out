GDTEST_RUNTIME_ERROR
>> SCRIPT ERROR
>> on function: test()
>> runtime/errors/for_key_value_not_dictionary.gd
>> 6
>> Unable to iterate "Array" as a key-value pair.
